src/view/testing.java

```diff
@@ -0,0 +1,5 @@
+package view;
+
+public class testing {

```

Comment:
Same thing, no point of having a empty testing class

Reply:
deleted at later issue.